openedx/core/djangoapps/api_admin/utils.py

```diff
@@ -1,5 +1,8 @@
 """ Course Discovery API Service. """
+from django.conf import settings
+
 from edx_rest_api_client.client import EdxRestApiClient
+from openedx.core.djangoapps.theming import helpers
```

Comment:
NIT/Not-your-code: Seems a little strange to me to be pulling a generic `get_value` from the theming app.  Probably could be refactored to `openedx.core.lib` somewhere